composer/callbacks/global_straggler_detector.py

```diff
@@ -0,0 +1,873 @@
+from megatron.core.utils import *
```

Comment:
May consider adding a link to where the megatron core file exists and a link to the original README for future documentation purposes.